### PR TITLE
add custom endpoints and localisation

### PR DESCRIPTION
**Pull request by wi-wissen**
support custom endpoints
set language by app localisation
add description for self hosted js
add error localisations
update cdn js version to 0.9.9
support single error attribute (occours in self hosted server with wrong api key)

custom endpoints are usefull for endpoint in europe or self hosted server

### CODE REVIEW

**Comment by ossycodes** [comment]
@wi-wissen what about the tests ?

File: src/FriendlyCaptcha.php [hunk 1]
```diff
@@ -161,7 +173,9 @@ public function verifyResponse($solution)
      */
     protected function sendRequestVerify(array $data = [])
     {
-        $response = $this->http->request('POST', static::VERIFICATION_API_ENDPOINT, [
+        //dd($data);
```
Comment (ossycodes) [inline src/FriendlyCaptcha.php]: @wi-wissen please clean up (i mean remove the dd($data))

**Comment by wi-wissen** [comment]
I did clean up.
I'm afraid I don't know how to write test cases here.

**Comment by ossycodes** [comment]
@wi-wissen in that case can you send me screenshots showing your changes all work fine please ?

**Comment by ossycodes** [comment]
@wi-wissen thank you for these fantastic improvements!

**Comment by wi-wissen** [comment]
Thanks for merging and adapting the tests. 🙂

**Pull request merged**